package-lock.json

```diff
@@ -1,8 +1,19472 @@
 {
   "name": "@guardian/cdk",
   "version": "4.0.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
I think you're on NPM v7? You'll want to downgrade to v6, I think . See https://github.com/guardian/cdk/pull/340 and https://github.com/guardian/cdk/pull/335#issuecomment-802678168.

Reply:
D'oh, thanks! Resolved in a coming push.